src/main/java/com/pinterest/secor/io/impl/AvroParquetFileReaderWriterFactory.java

```diff
@@ -108,9 +113,21 @@ public AvroParquetFileWriter(LogFilePath logFilePath, CompressionCodec codec) th
             CompressionCodecName codecName = CompressionCodecName
                     .fromCompressionCodec(codec != null ? codec.getClass() : null);
             topic = logFilePath.getTopic();
+
+            Schema schema;
```

Comment:
Does this logic also need to be replicated in AvroParquetFileReader?

Reply:
Done